### PR TITLE
fix(doc): fix issue with table rendering

### PR DESCRIPTION
**Pull request by amottier**
fix(doc): fix issue with table rendering

In getting started page, table that list data model was not displayed properly: content column shift on the right.
Remove leading spaces characters should fix the issue.

### CODE REVIEW

File: md/getting-started-tutorial.md [hunk 1]
```diff
@@ -175,16 +175,16 @@ To define the business data model:
...
+| Variable  | Data type | Mandatory |
+| --------- | --------- | -------- |
+| userId  | Long  | yes  |
+| departureDate  | Date  | yes  |
+| numberOfNights  | Integer  | yes  |
+| hotelNeeded  | Boolean  | yes  |
+| destination  | String  | yes  |
+| reason  | String  | yes  |
+| status  | String  | yes  |
+| refusalReason  | String  | no  |
```
Comment (redboul) [inline md/getting-started-tutorial.md]: Is it normal that the table is not aligned with the ordered list ?

Comment (amottier) [reply md/getting-started-tutorial.md]: The extra two spaces does not align the table with the ordered list items but lead to an issue in columns content display on our web site. Easy fix is to remove the two spaces.

**Comment by amottier** [comment]
Is there anything that still prevent merge of this modification? [Documentation in production on the website is not usable](http://documentation.bonitasoft.com/?page=getting-started-tutorial#toc1).

File: md/getting-started-tutorial.md (hunk 1 above)
Comment (redboul) [reply md/getting-started-tutorial.md]: Actually, you need three spaces to be aligned with the content above. 
Can you change this plz?
there may be some other tables and text that can be aligned too.

**Pull request merged**